scripts/make_openmic2018_index.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+import argparse
+from glob import glob
+from tqdm import tqdm
+import json
+import os
+from pathlib import Path
+from mirdata.validate import md5
+
+DATASET_INDEX_PATH = "../mirdata/datasets/indexes/openmic2018_index.json"
+
+
+def make_dataset_index(dataset_data_path):
+    path = Path(dataset_data_path)
+
+    # top-key level metadata
+    metadata_checksum = md5(path / Path("openmic-2018-metadata.csv"))
```

Comment:
It is only a code-style comment, but we use `os.path.join(...)` in the rest of the library. But it is the same... I think

Reply:
yes, it is the same, and a bit cleaner this way IMO.  Happy to change, but my understanding is that pathlib (which is python standard lib now) is the more modern way to do it.

Reply:
perfect for me